google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -246,9 +251,6 @@ private void checkSyncStashState() {
       mySyncStashCheckbox.setText(
           GctBundle.getString("clouddebug.stash.local.changes.and.sync", mySyncResult.getTargetSyncSHA().substring(0, 7)));
       mySyncStashCheckbox.setSelected(true);
-      myWarningLabel.setVisible(false);
```

Comment:
these are not good label names aye. We should name them by the message they display.